rust/libceed/src/operator.rs

```diff
@@ -2105,7 +2088,7 @@ impl<'a> Operator<'a> {
         let mut ptr_coarse = std::ptr::null_mut();
         let mut ptr_prolong = std::ptr::null_mut();
         let mut ptr_restrict = std::ptr::null_mut();
-        let ierr = unsafe {
+        self.op_core.check_error(unsafe {
```

Comment:
In the future (definitely a different PR), this could be `let (op_coarse, op_prolong, op_restrict) = ceed.op_core.check_error_with_3(|ptr_coarse, ptr_prolong, ptr_restrict| unsafe { ... })?;`.

Note that `Operator::from_raw` should be `unsafe` because it will dereference a raw pointer. Just noticed that while browsing (also not for this PR, and I'm not trying to snipe you into working on it).